configuration/micro_kernel_trait.rst

```diff
@@ -236,7 +236,7 @@ add a service conditionally based on the ``foo`` value::
         public function loadExtension(array $config, ContainerConfigurator $container, ContainerBuilder $builder): void
         {
             if ($config['foo']) {
-                $container->set('foo_service', new \stdClass());
+                $builder->set('foo_service', new \stdClass());
```

Comment:
I think this should either be

```php
$container->services()->set('foo_services', \stdClass::class);
```

or

```php
$builder->register('foo_service', \stdClass::class);
```

Reply:
@qzminski if you have some time, please make any of the changes suggested by @xabbuh so we can merge your contribution. Thanks!

Reply:
There you go :)